quickstarts/microsoft.kubernetes/aks-azure-linux-osguard/azuredeploy.parameters.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "clusterName": {
+        "value": "GEN-UNIQUE"
+      },
+      "dnsPrefix": {
+        "value": "GEN-UNIQUE"
+      },
+      "linuxAdminUsername": {
```

Comment:
I think we can drop linuxAdminUsername and sshRSAPublicKey from here since they aren't fields in the arm template.